packages/pagebuilder/lib/ContentTypes/Products/products.js

```diff
@@ -152,7 +149,7 @@ const Products = props => {
                     centerModeSmallClass
                 ].join(' ')}
             >
-                <SlickSlider {...carouselSettings}>{galleryItems}</SlickSlider>
+                <Carousel settings={carouselSettings} items={items} />
```

Comment:
Multiple Pagebuilder Products components could be rendered on a page. Potential race condition.